### DISCUSSION TITLE
Accessing the API from devices other than the container host

### DISCUSSION THREAD

**Discussion by maxcareyf** [TransformerOptimus/SuperAGI#359] [Q&A]
I've been tinkering with the project along with someone I live with, and we both agreed it'd be great to be able to access the app from any device within our private network. I've added the host machine's IP address to the list of allowed CORS addresses in main.py, but I can't seem to get past the "Initializing SuperAGI" page. I can interact with the GUI just fine, but the back-end isn't letting me interact with it from any device that isn't the container host. What else could I be missing to make this work?

**Comment by jonny255** [comment]
Would you mind telling me a bit more of your overall setup/infrastructure? For example, I've learned from experience that if you're running a docker container which uses it's own internal IP/subnet, you wouldn't be able to natively access it from outside the host computer without adjusting the network settings in docker or assigning a LAN IP to the container.
